packages/web-api/src/WebClient.ts

```diff
@@ -64,10 +64,6 @@ export enum WebClientEvent {
   RATE_LIMITED = 'rate_limited',
 }
 
-export interface WebAPICallOptions {
```

Comment:
Removing this interface makes the arguments to each API method now 'type safe' (more or less) in that developers can't provide whatever arguments they want anymore (at least not without overriding/ignoring TypeScript)